contributors.txt

```diff
@@ -1,46 +1,51 @@
-    10	Matt Palathingal <mattjose@umich.edu>
-     9	Andrew Karas <askaras@umich.edu>
-     9	Mike Henry <mikehenry@boisestate.edu>
-     8	Fengyi Gao <gaofy@umich.edu>
-     8	Ryan Marson <rmarson@umich.edu>
-     6	Allen LaCour <alacour@umich.edu>
-     6	Pengji Zhou <zhoupj@umich.edu>
-     6	Tom Grubb <tgrubb@umich.edu>
-     6	Yina Geng <yinageng@umich.edu>
-     5	Alex Dutton <alxdttn@umich.edu>
-     5	Carolyn Phillips <phillicl@umich.edu>
-     4	Ben Swerdlow <benswerd@umich.edu>
-     4	James Antonaglia <jamesaan@umich.edu>
-     4	Mayank Agrawal <amayank@umich.edu>
-     4	William Zygmunt <wzygmunt@umich.edu>
-     3	Greg van Anders <grva@umich.edu>
-     3	James Proctor <jproc@umich.edu>
-     3	Rose Cersonsky <rosecers@umich.edu>
-     3	Wenbo Shen <shenwb@umich.edu>
-     2	Maya Martirossyan <mmm457@cornell.edu>
-     1	Jiwoong Yu <jiwoongs0510@gmail.com>
-     1	Patrick Lawton <plawton@umich.edu>
-     1	Paul Dodd <pdodd@umich.edu>
+2122	Vyas Ramasubramani <vramasub@umich.edu>
+2040	Bradley Dice <bdice@bradleydice.com>
+1030	Eric Harper <harperic@umich.edu>
+456	Jin Soo Ihm <jinihm@umich.edu>
+294	Joshua A. Anderson <joaander@umich.edu>
+240	Matthew Spellings <mspells@umich.edu>
+166	Kelly Wang <kelwang@umich.edu>
+110	Erin Teich <erteich@umich.edu>
+79	Brandon Butler <butlerbr@umich.edu>
+66	M. Eric Irrgang <eirrgang@umich.edu>
+57	Charlotte Shiqi Zhao <zshiqi@umich.edu>
+53	Chrisy Du <xiyudu@umich.edu>
+51	dependabot <dependabot[bot]@users.noreply.github.com>
+41	Yezhi Jin <jinyezhi@umich.edu>
+40	Antonio Osorio <aosorio@umich.edu>
+35	Michael Stryk <mstryk@umich.edu>
+34	Tommy Waltmann <tomwalt@umich.edu>
+31	Richmond Newman <newmanrs@umich.edu>
+25	Carl Simon Adorf <csadorf@umich.edu>
+22	Jens Glaser <jsglaser@umich.edu>
+19	Andrew Kerr <andrewghkerr@gmail.com>
+19	Tim Moore <mtimc@umich.edu>
+14	pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>
+13	Pavel Buslaev <pbuslaev@gmail.com>
+10	Benjamin Schultz <baschult@umich.edu>
+10	Bryan VanSaders <bvansade@umich.edu>
+10	Matt Palathingal <mattjose@umich.edu>
+9	Andrew Karas <askaras@umich.edu>
+9	Mike Henry <mikehenry@boisestate.edu>
+8	Fengyi Gao <gaofy@umich.edu>
+8	Ryan Marson <rmarson@umich.edu>
+6	Allen LaCour <alacour@umich.edu>
+6	James Proctor <jproc@umich.edu>
+6	Pengji Zhou <zhoupj@umich.edu>
+6	Tom Grubb <tgrubb@umich.edu>
+6	Yina Geng <yinageng@umich.edu>
+5	Alex Dutton <alxdttn@umich.edu>
+5	Carolyn Phillips <phillicl@umich.edu>
+4	Ben Swerdlow <benswerd@umich.edu>
+4	Charlottez112 <56694726+Charlottez112@users.noreply.github.com>
+4	James Antonaglia <jamesaan@umich.edu>
+4	Mayank Agrawal <amayank@umich.edu>
+4	William Zygmunt <wzygmunt@umich.edu>
+3	Greg van Anders <grva@umich.edu>
+3	Rose Cersonsky <rosecers@umich.edu>
+3	Wenbo Shen <shenwb@umich.edu>
+2	Maya Martirossyan <mmm457@cornell.edu>
+1	Jiwoong Yu <jiwoongs0510@gmail.com>
+1	Patrick Lawton <plawton@umich.edu>
+1	Paul Dodd <pdodd@umich.edu>
+1	esiew <esiew@umich.edu>
```

Comment:
Not sure if it matters but for consistency with the rest of the list should this show my full name instead of my uniqname? (Emily Siew)

Reply:
@emilybwsiew Sure! We'll fix that. You should set your git configuration on all the computers you use to contribute code:

```bash
git config --global user.name "Emily Siew"
git config --global user.email "esiew@umich.edu"
```

Reply:
Ah right, I forgot to change it from when I had to set it to my uniqname for a class. Thank you!